includes/classes/Indexable/User/User.php

```diff
@@ -728,6 +728,9 @@ public function query_db( $args ) {
 			$args['order'] = 'desc';
 		}
 
+		$orderby_args = sanitize_sql_orderby( "{$args['orderby']} {$args['order']}" );
```

Comment:
I guess this duplicates the lines below. We should drop those bellow.

Reply:
I am afraid I don't understand what you mean :-(

As far as I can see, these lines come directly from the upstream:
https://github.com/10up/ElasticPress/blob/3.6.5/includes/classes/Indexable/User/User.php#L731
https://github.com/10up/ElasticPress/blob/3.6.5/includes/classes/Indexable/User/User.php#L732

Could you please clarify?

Reply:
Done